src/config/software-documentation/index.md

```diff
@@ -0,0 +1,13 @@
+# Software documentation
```

Comment:
I'm not in love with this name. Maybe `package documentation`?

Reply:
Done.